### ISSUE TITLE
Illegal copy of code with non MIT license being distributed

### ISSUE THREAD

**Issue by alerque** [dylanaraps/neofetch#1811]
Just a heads up, it looks like somebody has [copied this repo](https://github.com/The-Repo-Club/bfetch) in a way that is not allowed by the MIT including slapping a different license on the copy. Might want to look into what you can do...

**Comment by dylanaraps** [comment]
Thanks for the heads up.

https://github.com/The-Repo-Club/bfetch/issues/1

**Comment by alerque** [comment]
They just deleted the issue from their repo. I think this offender is aware of their problem, from poking around their repos they are almost all plagiarized with replaced licenses and have been previously posted under now defunct GitHub accounts. I suspect they got bounced before. I think as the project owner GitHub will facilitate a DCMA take-down on the repo because of the clearly illegal license.

**Comment by dylanaraps** [comment]
They *have* fixed the issue and restored the original license (though they also removed their name as well). I don't know if this will  be reverted sometime in the future though. Time will tell. I'll stay on top of it. Thanks again for letting me know.

**Comment by FabioLolix** [comment]
He have done it again https://github.com/The-Repo-Club/bfetch/commit/f83026e65820288f3cc2f08242f30afd95a40566

**Comment by dylanaraps** [comment]
...

Thanks for letting me know.

**Comment by dylanaraps** [comment]
https://github.com/The-Repo-Club/bfetch/issues/1

(They also clear the bug tracker each time...)

**Comment by alerque** [comment]
Yes, they have a whole scripted procedure for coping repos including the default license washing. When challenged they've been changing the license back to match upstream, then waiting a few weeks and resetting the repositories with the license washing back in place. This is not a good-faith contributor that made a mistake, this has been ongoing an repeated offenses even with the same projects (and under more than one GitHub alias). I do think having some DCMA filings would help GitHub deal with things closer to the source rather than playing whack-a-mole with their "projects".

**Comment by SomethingGeneric** [comment]
This mfer doesn't even have issues open now lmaoooo
